component/src/action_handler/actions/undelegate_claim.rs

```diff
@@ -42,7 +43,7 @@ impl ActionHandler for UndelegateClaim {
             .compounded_penalty_over_range(
                 &self.body.validator_identity,
                 self.body.start_epoch_index,
-                self.body.end_epoch_index,
+                end_epoch_index,
             )
```

Comment:
@avahowell per your question in discord: this is where the penalty is computed -- all that matters here is what slashings occurred during the epoch, not how many there were.